packages/common/src/Adaptor.js

```diff
@@ -196,41 +196,6 @@ export function lastReferenceValue(path) {
   return sourceValue(lastReferencePath);
 }
 
-/**
- * Scopes an array of data based on a JSONPath.
- * Useful when the source data has `n` items you would like to map to
- * an operation.
- * The operation will receive a slice of the data based of each item
- * of the JSONPath provided.
- * @public
- * @function
- * @example
- * map("$.[*]",
- *   create("SObject",
- *     field("FirstName", sourceValue("$.firstName"))
- *   )
- * )
- * @param {string} path - JSONPath referencing a point in `state.data`.
- * @param {function} operation - The operation needed to be repeated.
- * @param {State} state - Runtime state.
- * @returns {State}
- */
-export const map = curry(function (path, operation, state) {
```

Comment:
@josephjclark i think we should remove `import curry from 'lodash/fp/curry.js';` since it's no longer used

Reply:
Oh great catch, thanks!